### PR TITLE
Http request

### PR DESCRIPTION
**Pull request by colleend**
note: protocol for options is normally http by default.

### CODE REVIEW

File: problem-based-packs/insecure-transport/js-node/http-request.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+function bad_http() {
+    // ruleid: http-request
+    http.get('http://nodejs.org/dist/index.json', (res) => {
+    const { statusCode } = res;})
+
+    // ruleid: http-request
+    https.get('http://nodejs.org/dist/index.json', (res) => {
```
Comment (minusworld) [inline problem-based-packs/insecure-transport/js-node/http-request.js:7]: Does this actually go over HTTP? 😱

Comment (colleend) [reply problem-based-packs/insecure-transport/js-node/http-request.js:7]: you're totally right!! I don't think it actually goes over http :o

**Review by minusworld — APPROVED** [review]

File: problem-based-packs/insecure-transport/js-node/http-request.js (hunk 1 above)
Comment (minusworld) [reply problem-based-packs/insecure-transport/js-node/http-request.js:7]: We could test using wireshark 🤔   🦈

Comment (colleend) [reply problem-based-packs/insecure-transport/js-node/http-request.js:7]: thats a good idea!!

Comment (colleend) [reply problem-based-packs/insecure-transport/js-node/http-request.js:7]: I'll first push this, if requests to 'http...' through the https module actually doesn't encrypt traffic, i'll update it back to the OG 💯

Comment (colleend) [reply problem-based-packs/insecure-transport/js-node/http-request.js:7]: tested -- node actually throws "TypeError [ERR_INVALID_PROTOCOL]: Protocol "http:" not supported. Expected "https:" error if you attempt to request an http website.

**Pull request merged**